README.md

```diff
@@ -2025,6 +2025,7 @@ Before hosting one, please see [shortcomings](https://en.wikipedia.org/wiki/URL_
 
 - [Blink](https://docs.blink.rest) - Easy-to-host, SSO-integrated, CDN-powered link shortener (+decoupled analytics) for teams. ([Source Code](https://github.com/JaneJeon/blink)) `AGPL-3.0` `Nodejs`
 - [Clink](https://short.swurl.xyz/) - Dead-simple, super minimal URL shortener in C. ([Source Code](https://git.swurl.xyz/swirl/clink)) `AGPL-3.0` `C`
+- [Easyshortener](https://github.com/easypanel-community/easyshortener) - A simple URL shortener written in PHP using Laravel 10 `MIT` `PHP`
```

Comment:
```suggestion
- [Easyshortener](https://github.com/easypanel-community/easyshortener) - A simple URL shortener. `MIT` `PHP/Nodejs`
```

> The Language tag is the main server-side requirement for the software.
>> https://github.com/easypanel-community/easyshortener#installation
>>  composer install
>> npm run build

-> `PHP/Nodejs`

 > considering it has docker

Undocumented.

Reply:
Docker seems to be documented - are you sure it's a good idea to leave it out?

Reply:
Now it is documented ;) I edited my suggestion above, please check and apply it.